main/migrations/0007_auto_20151115_1833.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not sure why your commit contains so many migration files.
I will check on my machine to try and figure out what's happening